app/src/main/res/layout/test_activity.xml

```diff
@@ -724,6 +724,19 @@
 
             <!-- Errors -->
             <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textAppearance="?android:attr/textAppearanceSmall"
+                android:text="Answers can be changed even after submission."
```

Comment:
Please store the string in https://github.com/Ana06/medical-data-android/blob/master/app/src/main/res/values/strings.xml

This allow us to translate the app to other languages easily. The Spanish translations are stored here:

https://github.com/Ana06/medical-data-android/blob/master/app/src/main/res/values-es/strings.xml

It is not necessary that you update the Spanish strings when you add a text, as you may not speak Spanish. But in case that you add that one the translation could be:

_Las respuestas pueden modificarse más adelante._